Makefile

```diff
@@ -0,0 +1,17 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+lint:
```

Comment:
Not sure where this file came from but I don't think it's supposed to be here

Reply:
Just wanted to add a local lint/style check module, but @atilag informed me that we're moving to tox. I'll remove this.